noxfile.py

```diff
@@ -0,0 +1,110 @@
+"""
+nox configuration for cookiecutter project template.
+"""
+
+import contextlib
+import os
+import tempfile
+from pathlib import Path
+
+import nox
+
+CI = os.environ.get('CI') is not None
+
+ROOT = Path('.')
+PYTHON_VERSIONS = ['3.9']
+PYTHON_DEFAULT_VERSION = PYTHON_VERSIONS[-1]
+
+nox.options.default_venv_backend = 'venv'
+nox.options.reuse_existing_virtualenvs = True
+
+
+# In CI, use Python interpreter provided by GitHub Actions
+if CI:
+    nox.options.force_venv_backend = 'none'
+
+
+CRUFTED_PROJECT_TMPDIR = None
+
+MD_PATHS = ['*.md']
+
+
+def run_readable(session, mode="fmt"):
+    session.run(
+        'docker',
+        'run',
+        '--rm',
+        '-v', f'{ROOT.absolute()}:/data',
+        '-w', '/data',
+        '-u', f'{os.geteuid()}:{os.getegid()}',
+        'ghcr.io/bobheadxi/readable:v0.5.0@sha256:423c133e7e9ca0ac20b0ab298bd5dbfa3df09b515b34cbfbbe8944310cc8d9c9',
+        mode, *MD_PATHS
+    )
+
+
+@nox.session(name='format', python=PYTHON_DEFAULT_VERSION)
+def format_(session):
+    """Lint the code and apply fixes in-place whenever possible."""
+    session.run('pip', 'install', '-e', '.[format]')
+    session.run('ruff', 'check', '--fix', '.')
+    run_readable(session, mode="fmt")
+
+
+@nox.session(python=PYTHON_DEFAULT_VERSION)
+def lint(session):
+    """Run linters in readonly mode."""
+    session.run('pip', 'install', '-e', '.[lint]')
+    session.run('ruff', 'check', '--diff', '.')
+    run_readable(session, mode="check")
```

Comment:
Is there any sense of running checks on non-crufted project? Cause it will cause errors where template tags are present, for example:
```
error: Failed to parse {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/{{cookiecutter.django_default_app_name}}/tasks.py:1:2: unexpected token '%'
error: Failed to parse {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/{{cookiecutter.django_default_app_name}}/metrics.py:1:2: unexpected token '%'
```

Reply:
it doesn't which is why I excluded it in config:
https://github.com/reef-technologies/cookiecutter-rt-django/pull/114/files/85372c943bdccac053616486772f7527b022a963#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R30-R32